tests-integration/tests/http.rs

```diff
@@ -511,7 +511,9 @@ pub async fn test_config_api(store_type: StorageType) {
     enable_memory_catalog = false
     rpc_addr = "127.0.0.1:3001"
     rpc_runtime_size = 8
-    heartbeat_interval_millis = 5000
+
+    [heartbeat]
+    interval_millis = 5000
```

Comment:
@Kree0 looks like one more adjustment is required for this test.
To reproduce the failure you can use `cargo test test_config_api` command